playground/html/__tests__/html.spec.ts

```diff
@@ -250,9 +242,3 @@ describe.runIf(isServe)('invalid', () => {
     expect(content).toBeTruthy()
   })
 })
-
-test('importmap', () => {
-  expect(browserLogs).not.toContain(
-    'An import map is added after module script load was triggered.'
-  )
-})
```

Comment:
Is this importmap test removed intentionally? I think we'd still want to check for this.

Reply:
Weird, looks like the merge went wrong 🤔